### PR TITLE
Add v5.3 TPCH report

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add v5.3 TPCH report

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7540
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@shichun-0415 Please fix the Jenkins failure and link this PR to the related one that adds the alias.

File: benchmark/v5.3-performance-benchmarking-with-tpch.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
-title: TiDB TPC-H 100GB Performance Test Report -- v5.2 MPP mode vs. Greenplum 6.15.0 and Apache Spark 3.1.1
-summary: Compare the TPC-H 100GB performance of TiDB 5.2 MPP mode, Greenplum 6.15.0, and Apache Spark 3.1.1.
+title: TiDB TPC-H 100GB Performance Test Report -- v5.3 MPP mode vs. Greenplum 6.15.0 and Apache Spark 3.1.1
```
Comment (shichun-0415) [inline benchmark/v5.3-performance-benchmarking-with-tpch.md:2]: The alias is added in https://github.com/pingcap/docs/pull/6866.

Comment (shichun-0415) [inline benchmark/v5.3-performance-benchmarking-with-tpch.md:2]: The alias is added in #6866.

**Review by ilovesoup — APPROVED** [review]
lgtm

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**